lms/static/sass/views/_verification.scss

```diff
@@ -821,6 +829,114 @@
 
     // indiv slides - review
     #wrapper-review {
+      color: $black;
+
+      .page-title {
+          border-bottom: 2px solid $darkGrey;
+          padding-bottom: 15px;
+          margin-bottom: 20px;
+          font-weight: bold;
+          text-transform: inherit;
+        }
+
+      .review {
+
+      .certificate {
```

Comment:
Indentation issue starting here, review and certificate are on the same line. Should be updated to reflect proper nesting from this point onward.